main-settings/src/main/scala/sbt/std/InputWrapper.scala

```diff
@@ -182,12 +182,12 @@ object ParserInput {
       }
     }
 
-  private def wrapInputTask[T](c: Context)(tree: c.Tree, pos: c.Position) = {
+  private def wrapInputTask[T: c.WeakTypeTag](c: Context)(tree: c.Tree, pos: c.Position) = {
```

Comment:
This is a fixup commit, to be folded into the original, faulty refactor. The original code already used WeakTypeTag, so there's no semantic change, see https://github.com/sbt/sbt/pull/2655/files#diff-407b2912ff3a899d0ed98050a9b91908R176.